### DISCUSSION TITLE
Reply to stories

### DISCUSSION THREAD

**Discussion by IsaacHatton** [dclstn/better-chat#435] [Ideas]
As requested here: https://discord.com/channels/1217438562107002940/1217438562107002943/1239996065944440843
> Imma put it there later but say it now cuz ill forget so is it possible to make it so i can reply to stories on pc

Being able to reply to stories from the web app would be helpful.

**Comment by dclstn** [comment]
This seems like a native snapchat feature, I don't think we'd ever add this to BetterSnap
